src/utils/function.js

```diff
@@ -24,7 +24,7 @@
  * @param {number} minInterval the minimum time interval in millisecond
  * @returns {function()}
  */
-export function rateLimit(win, callback, minInterval) {
```

Comment:
`utils/function.js` sounds like a too broad name, and eventually become a garbage bin for everything.